packages/serverless-contracts/package.json

```diff
@@ -71,6 +71,7 @@
     "concurrently": "^7.6.0",
     "dependency-cruiser": "^12.1.0",
     "eslint": "^8.28.0",
+    "http-status-codes": "^2.2.0",
```

Comment:
This should be listed in `dependencies`, as it will be used in the exported types.
I think there are other` devDependencies` that should be moved up also, can you check and move them too?

Reply:
Given that, you can also change the commit message to `feat(serverless-contracts): ...` so that it will be in the correct place in the release notes